### PR TITLE
Add uavro

### PR DESCRIPTION
**Pull request by martindurant**

### CODE REVIEW

File: recipes/uavro/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - numpy
```
Comment (synapticarbors) [inline recipes/uavro/meta.yaml]: See:

https://conda-forge.org/docs/meta.html#building-against-numpy

File: recipes/uavro/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    - fastparquet
+
+test:
+  imports:
+    - uavro
+
+about:
+  home: https://github.com/martindurant/uavro
+  license: MIT
+  license_family: MIT
```
Comment (synapticarbors) [inline recipes/uavro/meta.yaml]: please include the `license_file` manually (https://conda-forge.org/docs/meta.html#packaging-the-license-manually). See also:

https://github.com/martindurant/uavro/issues/6

File: recipes/uavro/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
```
Comment (synapticarbors) [inline recipes/uavro/meta.yaml]: add `toolchain` as a `build` requirement.

**Comment by martindurant** [comment]
@synapticarbors - finally passed; all OK now?

**Pull request merged**